kafo.gemspec

```diff
@@ -26,12 +26,19 @@ Gem::Specification.new do |spec|
   # puppet module parsing
   spec.add_dependency 'kafo_parsers'
   spec.add_dependency 'puppet', '< 4.0.0'
-  # better logging
-  spec.add_dependency 'logging', '< 2.0.0'
   # CLI interface
   spec.add_dependency 'clamp', '>= 0.6.2'
-  # interactive mode
-  spec.add_dependency 'highline', '>= 1.6.21', '< 1.7'
+  if RUBY_VERSION >= '1.9'
```

Comment:
You'll need to add any pinnings for 1.8 in a Gemfile (so it only affects development/test) rather than the gemspec, since when it's built as a binary gem then these conditionals won't stay in the final gem.